static_src/components/app_quicklook.jsx

```diff
@@ -56,7 +48,7 @@ export default class AppQuicklook extends React.Component {
     const statusClass = !isHealthyApp(app) && 'status-error';
 
     return (
-      <a className={ this.styler(statusClass) } href={ this.appHref() }>
+      <a className={ statusClass } href={ this.appHref() }>
```

Comment:
Use `classNames` inline? Makes the semantics of `!isHealthyApp(app) && 'status-error'` clearer:

```jsx
<a className={classNames({ 'status-error': !isHealthyApp(app) })} />
```